app/controllers/admin/attachments_controller.rb

```diff
@@ -8,6 +8,8 @@ class Admin::AttachmentsController < Admin::BaseController
 
   def index; end
 
+  def reorder; end
```

Comment:
How is the current page being rendered?

Reply:
There is no current page, the reordering functionality is in the index page.
Let me know if there's any changes you'd suggest making to the PR description or commits to make that more clear.

Reply:
@dnkrj I don't see this section on the index page. I think it would be useful to describe more fully how it is accessed.